arm_robots/src/arm_robots/base_robot.py

```diff
@@ -96,3 +110,34 @@ def check_inputs(self, group_name: str, ee_link_name: str):
         if group_name not in groups:
             rospy.logwarn_throttle(1, f"Group [{group_name}] does not exist. Existing groups are:")
             rospy.logwarn_throttle(1, groups)
+
+    def create_cartesian_impedance_controller(self, motion_status_listeners: List[Listener],
```

Comment:
formatting is odd here. I'd prefer _all_ parameters on a new line, so add a newline between `self` and `motion...`